sdk/core/azure-common/dev_requirements.txt

```diff
@@ -1,2 +1,3 @@
 -e ../../../tools/azure-sdk-tools
 -e ../azure-core
+msrestazure
```

Comment:
Why does azure-common require msrestazure?

Reply:
Tests. The older get_credentials hits it.